site/_posts/2020-09-01-microservices-gateway.md

```diff
@@ -0,0 +1,267 @@
+---
+layout: blog-post
+title: Building a microservices gateway application 
```

Comment:
Weave in "Auth". Maybe "Authenticating with a microservices gateway application"